src/constants/routers.ts

```diff
@@ -3,4 +3,15 @@ export const ROUTESTRING = {
   RECORDING: 'recording',
   NOTIFICATION: 'notification',
   PROFILE: 'profile',
+  LOGIN: 'login',
+  SIGNUP: 'signup',
+};
+
+export const ROUTEURL = {
```

Comment:
```suggestion
export const ROUTE_URL = {
```

이것도 이렇게 구분자를 두는게 더 가독성에 도움이 될 것 같습니다!